msvc/sdk-build.props

```diff
@@ -90,7 +92,7 @@
       <RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>
       <RuntimeLibrary Condition="'$(Configuration)'=='Debug'">MultiThreadedDebugDLL</RuntimeLibrary>
       <SystemHeaderDeps>true</SystemHeaderDeps>
-      <AdditionalOptions>-Werror -Wno-microsoft %(AdditionalOptions)</AdditionalOptions>
+      <AdditionalOptions>-Werror -Wno-microsoft -Wno-c++1z-extensions -Wno-ignored-attributes -Wno-deprecated-declarations %(AdditionalOptions)</AdditionalOptions>
```

Comment:
Aha, a better solution: clang will not generate warnings from headers that it considers "system" headers. So if we add --system-header-prefix=winrt/ to the command line, we get no more warnings from the C++/WinRT headers.

---
In reply to: [109240292](https://github.com/Microsoft/WinObjC/pull/2382#discussion_r109240292) [](ancestors = 109240292,109239222)